packages/inter-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -46,14 +46,16 @@ import {
   setupElectorateReserveAndAuction,
 } from './vaultFactoryUtils.js';
 
-/** @typedef {Record<string, any> & {
+/**
+ * @typedef {Record<string, any> & {
  * aeth: IssuerKit & import('../supports.js').AmountUtils,
  * run: IssuerKit & import('../supports.js').AmountUtils,
  * bundleCache: Awaited<ReturnType<typeof unsafeMakeBundleCache>>,
  * rates: VaultManagerParamValues,
  * loanTiming: LoanTiming,
  * zoe: ZoeService,
- * }} Context */
+ * }} Context
```

Comment:
Non-blocking suggestion:
```suggestion
 * @typedef {import('./driver.js').DriverContext} FullContext
 * @typedef {Pick<FullContext, 'aeth' | 'run' | 'rates' | 'loanTiming' | 'zoe'> & {
 *   bundleCache: Awaited<ReturnType<typeof unsafeMakeBundleCache>>,
 *   rates: VaultManagerParamValues,
 *   loanTiming: LoanTiming,
 * }} Context
```
and it looks like `bundleCache` isn't even used.